### PR TITLE
Add flag to poltiezone & hulpverlenigzone besluitenlijst

### PR DESCRIPTION
**Pull request by aatauil**
## Description

When a submission is send, there is a background job checking if the send submission should be published tot eh publieke besluitendatabank. The prepare-submissions-for-export service is responsible for deciding if a submission should to to the publieke besluitendatabank based on a list of "rules". 

This PR does 2 things:

- update rules to include politiezone & hulpverleningszone submissions (https://github.com/lblod/prepare-submissions-for-export-service/pull/8)
- add migration to also export previously send in submission "retroactief" (this PR)

 ## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Other

 ## How to test

- clone prepare-submissions-for-export service
- checkout the feature/additions-besluitenlijst branch
- then in app-digitaal-loket you have to link through volumes:
 ```
 environment: 
   NODE_ENV: "development"
  volumes:
   - "path/to/prepare-submissions-for-export/:/app/"
 ```
 
 - Next you need to link the consumer-producer from app-digitaal-loket to app-public-decisions-database
( Development setup guide here: https://github.com/lblod/app-digitaal-loket/wiki/Extending-Delta-Producer )
 
 - next login to frontend loket as a politieraad and submit a toezicht form
 - check the logs of the delate-consumer on the app-public-decisions-database and see if it starts consuming the besluit (logs will start moving a lot)

 ## Links to other PR's

 - prepare-submissions-for-export: https://github.com/lblod/prepare-submissions-for-export-service/pull/8

### CODE REVIEW

File: config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+PREFIX prov:    <http://www.w3.org/ns/prov#>
+PREFIX ext:     <http://mu.semte.ch/vocabularies/ext/>
+PREFIX eli:     <http://data.europa.eu/eli/ontology#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX pav:     <http://purl.org/pav/>
+prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX schema:  <http://schema.org/>
+PREFIX dct:     <http://purl.org/dc/terms/>
+
+INSERT {
```
[the remark this answers is not in the document]
Comment (cecemel) [reply config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql:12]: High level, i would split in smaller migrations

File: config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+WHERE {
+  GRAPH ?g {
+    ?sub a meb:Submission ;
+        adms:status <http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c> ; # SENT
+        prov:generated ?form ;
+        pav:createdBy ?bestuurseenheid ;
+        dct:subject ?submissionDocument .
+
+
+    ?form <http://lblod.data.gift/vocabularies/besluit/authenticityType> |
```
Comment (cecemel) [inline config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql:40]: here you could make a separate list of types that need to be exported. That's a one time check you can do, not to check for every separate sbumission. (the walking of this tree is super expensive)

File: config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <http://mu.semte.ch/vocabularies/ext/regulationType> |
+          <http://mu.semte.ch/vocabularies/ext/decisionType> |
+          <http://mu.semte.ch/vocabularies/ext/taxType> |
+          <http://data.europa.eu/eli/ontology#passed_by>/<http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan>/<http://data.vlaanderen.be/ns/besluit#bestuurt>/<http://data.vlaanderen.be/ns/besluit#classificatie> ?concept.
+
+    ?form
+        ext:decisionType <https://data.vlaanderen.be/id/concept/BesluitDocumentType/3fa67785-ffdc-4b30-8880-2b99d97b4dee> ; # Besluitenlijst
+        eli:passed_by ?bestuursorgaan .
+
+    OPTIONAL { ?submissionDocument adms:status ?submissionDocumentStatus . }
```
Comment (cecemel) [inline config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql:50]: not sure why optional are needed, I think any of these optional fails (expect perhaps the attachments) the data is broken.

File: config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    ?bestuursorgaan mandaat:isTijdspecialisatieVan/besluit:classificatie ?bestuursorgaanClassificatieCode .
+    ?bestuurseenheid besluit:classificatie ?bestuurseenheidClassificatieCode .
+    }
+
+  GRAPH ?i {
+    ?concept <http://www.w3.org/2004/02/skos/core#inScheme> |
+             <http://www.w3.org/2004/02/skos/core#topConceptOf> ?conceptScheme.
+
+    FILTER ( ?conceptScheme IN (
```
Comment (cecemel) [inline config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql:71]: I think here they are interested in all submissions from the bestuursorganen, so maybe no extra filter needed?

**Review by cecemel — CHANGES_REQUESTED** [review]

**Comment by aatauil** [comment]
moved to https://github.com/lblod/app-digitaal-loket/pull/452

**Pull request closed without merging**